app/controllers/pages_controller.rb

```diff
@@ -1,12 +1,25 @@
 class PagesController < ApplicationController
-  # before_filter :authenticate_user!
-
   def home
-    if user_signed_in?
-      @user = current_user
-      authorize @user
+    return unless user_signed_in? # no additional logic needed if not signed in
+    authenticate_user!
+    @user = current_user
+    authorize @user
+    @taken_courses = TakenCourse.includes(:course).where(user_id: @user.id)
+  end
+
+  def planner # shows the active plan if set, otherwise shows new
+    authenticate_user!
+    # I couldn't get this to not load user a second time, so might as well
+    # just include it anyway
+    plan = Plan
+           .includes(:user, taken_courses: [:course], planned_courses: [:course])
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.